Lib/nntplib.py

```diff
@@ -310,8 +309,9 @@ class _NNTPBase:
     encoding = 'utf-8'
     errors = 'surrogateescape'
 
-    def __init__(self, file, host,
-                 readermode=None, timeout=_GLOBAL_DEFAULT_TIMEOUT):
+    def __init__(self, host, file=None,
```

Comment:
I don't want to add a new parameter `file` for NNTP.

Reply:
Ok, I will change this.